### DISCUSSION TITLE
species classes

### DISCUSSION THREAD

**Discussion by saskiad** [AllenNeuralDynamics/aind-data-schema#646] [General]
Since we are no longer using the PIDName construct, is there a reason to keep the abbreviation field for the species classes when we aren't going to use it? @jtyoung84

**Comment by jtyoung84** [comment]
I tried to follow the original branch wherever possible and so I had the Species class inherit from the PIDName classes. If abbreviations aren't expected to be used, we can remove the inheritance and the abbreviation field. It makes sense to do that.

**Comment by saskiad** [comment]
Cool - I'll add that to my PR. I get why it's there, but I think it'll just be cleaner since we won't use abbreviations here. Thanks!
